asserts/snap_asserts.go

```diff
@@ -160,6 +165,11 @@ type SnapRevision struct {
 	timestamp    time.Time
 }
 
+// Series returns the series of the snap submitted to the store.
```

Comment:
Pushed a couple of doc comment changes, including the initial suggestion.

I'll remove mentions of "snap build" in #675 to avoid conflicts, and because it actually makes more sense there.